### PR TITLE
3066 incorrect message while enabling disabling password

### PR DESCRIPTION
**Pull request by Sam-Kohl**
On Form.jsx line 14, removed 'Form' and replaced with 'Password'.

### CODE REVIEW

**Comment by gsmithun4** [comment]
@Sam-Kohl Can you please fix the lint errors

**Comment by Navaneeth-pk** [comment]
Please fix the lint errors and sign the CLA so that we can merge this PR.
@Sam-Kohl

**Comment by Sam-Kohl** [comment]
Hey all. Sorry for the delay. Was away for a wedding.
Fixed and committed.  CLA signed.
Thanks for your patience.

File: frontend/src/ManageSSO/Form.jsx [hunk 1]
```diff
@@ -11,9 +11,9 @@ export function Form({ settings, updateData }) {
         const enabled_tmp = !enabled;
         setEnabled(enabled_tmp);
         updateData('form', { id: data.id, enabled: enabled_tmp });
-        toast.success(`${enabled_tmp ? 'Enabled' : 'Disabled'} Form login`, {
+        toast.success(`${enabled_tmp ? 'Enabled' : 'Disabled'} Password login`), {
           position: 'top-center',
-        });
+        };
```
Comment (gsmithun4) [inline frontend/src/ManageSSO/Form.jsx:16]:
```
toast.success(`${enabled_tmp ? 'Enabled' : 'Disabled'} Password login`, {
    position: 'top-center',
});
```

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

**Review by Navaneeth-pk — APPROVED** [review]

**Pull request merged**